mods/ra/weapons/ballistics.yaml

```diff
@@ -160,29 +96,25 @@ TurretGun:
 			Heavy: 25
 			Concrete: 50
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
 	Warhead@3Eff: CreateEffect
 		Explosions: artillery_explosion
 		ImpactSounds: kaboom15.aud
-		InvalidImpactTypes: Water
 	Warhead@4EffWater: CreateEffect
 		Explosions: med_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
+
+155mm:
+	Inherits: ^Artillery
+	Projectile: Bullet
+		ContrailLength: 30
 
 8Inch:
+	Inherits: ^Artillery
```

Comment:
This introduces a new 3 cell minimum range for the cruiser.  I think this is a good thing (it firing into the cell next door looked bogus), but shouldn't sneak in unnanounced.